src/redux/books/books.jsx

```diff
@@ -0,0 +1,44 @@
+// Actions
+
+const ADD_BOOK = 'bookstore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
+
+// Reducer
+export default function reducer(state = [], action = {}) {
+  switch (action.type) {
+    case ADD_BOOK: {
+      return [
+        ...state,
+        {
+          title: action.title,
+          author: action.author,
+          id: action.id,
+        },
+      ];
+    }
+
+    case REMOVE_BOOK: {
+      return [...state].filter((item) => item !== action.id);
+    }
+
+    default: return state;
+  }
+}
+
+export function addBook(book) {
+  return {
+    type: ADD_BOOK,
+    title: book.title,
+    author: book.author,
+    id: book.id,
+  };
+}
+
+export function removeBook(book) {
+  return {
+    type: REMOVE_BOOK,
+    title: book.title,
+    author: book.author,
+    id: book.id,
```

Comment:
**OPTIONAL** - Since you will not be needing the title and author of the book when deleting I suggest passing only the `id` of the book to be deleted would suffice 👍. Please fix this 😄

Reply:
I wasn't sure if I should only pass the id to make it so that it would be a complete book. It makes sense to only pass the id. Thanks for pointing this out.